main.go

```diff
@@ -41,13 +42,15 @@ func releaseToStream(releaseArch *ReleaseArch, release Release) StreamArch {
 			Formats: releaseArch.Media.Azure.Artifacts,
 		}
 		artifacts.Azure = &azure
-		azureImage := StreamCloudImage{}
 
-		if releaseArch.Media.Azure.Images.Global != nil {
+		if releaseArch.Media.Azure.Images != nil &&
```

Comment:
The golang style of formatting makes this block a bit hard to read at a glance. Might read better doing something like so:
```golang
if az := releaseArch.Media.Azure.Images; az != nil && az.Global != nil && az.Global.Image != nil {
    ...
}
```

Reply:
Good idea, fixed it!